core/language/en-GB/Misc.multids

```diff
@@ -74,6 +77,10 @@ Shortcuts/Input/Tab-Left/Hint: Select the previous Tab
 Shortcuts/Input/Tab-Right/Hint: Select the next Tab
 Shortcuts/Input/Up/Hint: Select the previous item
 Shortcuts/SidebarLayout/Hint: Change the sidebar layout
+Switcher/Subtitle/theme: Switch themes
```

Comment:
I think all of these should perhaps be singular "Switch theme", "Switch layout" etc. I'm not sure if we should capitalise the second word, but I'd be inclined to if it doesn't look odd